cpg_infra/billing_aggregator/aggregate/hail.py

```diff
@@ -112,13 +112,14 @@ def get_finalised_entries_for_batch(batch: dict) -> List[Dict]:
 
         cost = utils.get_total_hail_cost(currency_conversion_rate, raw_cost=raw_cost)
         usage = resource_usage.get(batch_resource, 0)
+        # 2023-03-07 mfranklin: I know this key isn't unique, but to avoid issues
+        # with changing the resource_id again, we'll only use the batch_id as the key
+        # as it's sensible for us to assume that all the entries exist if one of the
+        # entries exists
```

Comment:
Is that really true though if some jobs are running much longer than others? Should the key at least also include the job ID?

Reply:
This one aggregates the whole batch (instead of per job), because the batch is completed + closed. 
We only distribute seqr (right now).